### ISSUE TITLE
Can not reverse() @list_route methods in ViewSet with ExtendedDefaultRouter.

### ISSUE THREAD

**Issue by bryson** [chibisov/drf-extensions#168]
drf-extensions 0.3.1
restframework 3.4.x, 3.5

Standard viewset, no nested routes, reverse for `@list_route` methods does not work:

`NoReverseMatch: Reverse for 'project-drafts' with arguments '()' and keyword arguments '{}' not found. 0 pattern(s) tried: []`

Switching the router back to `DefaultRouter` immediately gives the proper result:

```
In [1]: reverse('project-drafts')
Out[1]: '/api/v1/projects/drafts/'
```

**Comment by AndreasBackx** [comment]
The resolved name is `project-drafts-list` most probably, the naming for routes is [documented](http://chibisov.github.io/drf-extensions/docs/#nested-routes).

**Comment by bryson** [comment]
I'm not trying to `reverse` the `list` method, I'm trying to `reverse` an ad-hoc method with a `@list_route` decorator. In DRF, these methods are added to a ViewSet's automatically-generated routes when using the `DefaultRouter`. This is also [documented](http://www.django-rest-framework.org/api-guide/viewsets/#marking-extra-actions-for-routing). In DRF-extensions, these routes are not automatically generated, and can no longer be reversed.

**Comment by rpkilby** [comment]
This would be fixed by #214. Dynamic list routes would be consistent with DRF and drop the `-list` suffix.
